src/main/java/hudson/plugins/tics/AbstractApiCall.java

```diff
@@ -71,9 +71,6 @@ protected final CloseableHttpClient createHttpClient() throws KeyManagementExcep
                 final String proxyName = proxy.getName();
                 final int proxyPort = proxy.getPort();
                 final List<Pattern> noProxyPatterns = proxy.getNoProxyHostPatterns();
```

Comment:
Please replace by `final ImmutableList<Pattern> noProxyPatterns = ImmutableList.copyOf(proxy.getNoProxyHostPatterns());`

Reply:
Fixed in https://github.com/jenkinsci/tics-plugin/pull/28/commits/a66271079d7c5abd7e61140174044efb021cd579

Reply:
Fixed in https://github.com/jenkinsci/tics-plugin/pull/28/commits/a66271079d7c5abd7e61140174044efb021cd579